appstudio-controller/controllers/appstudio.redhat.com/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -272,6 +274,15 @@ func generateExpectedGitOpsDeployment(component appstudioshared.ComponentStatus,
 		}
 	}
 
+	res.ObjectMeta.Labels = make(map[string]string)
+
+	// Append ASEB labels with prefix "appstudio.openshift.io" to the gitopsDeployment labels
+	for bindingKey, bindingLabels := range binding.Labels {
+		if strings.HasPrefix(bindingKey, "appstudio.openshift.io") {
```

Comment:
I think the story criteria I originally specified is incorrect, lets change this to `strings.Contains` rather than `strings.HasPrefix`: it looks like HACBS folks would like _any match_ within the key name, rather than just a prefix match.